.github/workflows/ci-mysql.yml

```
@@ -0,0 +1,29 @@
+on:
+  - pull_request
+  - push
+
+name: ci-mysql
+
+jobs:
+  mysql:
+    name: MySQL database tests.
+    uses: ./.github/workflows/phpunit-database.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/phpunit-database.yml": open /home/jailuser/git/.github/workflows/phpunit-database.yml: no such file or directory

(workflow-call)